src/mock/index.ts

```diff
@@ -0,0 +1,42 @@
+export interface Project {
+  id: string;
+  name: string;
+  verified: boolean;
+  description: string;
+  tags: Array<string>;
+  links: {
+    facebook: {
+      id: string;
+      url: string;
+    };
+    discord: {
+      id: string;
+      url: string;
+    };
+    twitter?: {
+      id: string;
+      url: string;
+    };
+  };
+}
+
+export const projects: Array<Project> = [
+  {
+    id: "web3ph",
```

Comment:
```suggestion
    id: "web3phl",
```

Reply:
This is to prevent confusion with the other Web3Ph group that goes with "web3ph"

Reply:
This does make sense @berrycakes please use `web3phl` or `web3philippines` directly.